src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -415,6 +415,21 @@ export default function (AB, ibase) {
                   // get a new instance of a field:
                   field = this.CurrentObject.fieldNew(vals);
 
+                  if (
+                     vals.columnName !== field.columnName &&
+                     field.columnName === ""
+                  ) {
+                     this.AB.Webix.message({
+                        text: "The column name does not allow letters.",
```

Comment:
Sorry should have said the first time. Does not allow letters isn't a very good explanation. Maybe the column name can't contain special characters?

Reply:
Sure! thanks!